mwclient/client.py

```diff
@@ -970,8 +970,14 @@ def get_token(
         if self.version is None or self.require(1, 24, raise_error=False):
             # The 'csrf' (cross-site request forgery) token introduced in 1.24 replaces
             # the majority of older tokens, like edittoken and movetoken.
-            if type not in {'watch', 'patrol', 'rollback', 'userrights', 'login'}:
-                type = 'csrf'
+            if type not in {
+                'watch', 
+                'patrol', 
+                'rollback', 
+                'userrights', 
+                'login', 
+                'createaccount'}:
+                    type = 'csrf'
```

Comment:
@wadoli, thanks again! It looks like Flake8 still has an issue with this -- something like the following should work. 

Btw, you can check it locally by running `flake8 mwclient` from the repo root (`pip install flake8` if you don't have it installed).

```suggestion
            if type not in {
                'watch',
                'patrol',
                'rollback',
                'userrights',
                'login',
                'createaccount',
            }:
                type = 'csrf'
```

Reply:
also, once it passes, please squash to a single commit. thanks!

Reply:
Thank you for your hints and support and sorry for not reacting quicker. I was sick for the last two days. Also I have only limited git experience. I have now read and acted on https://mwclient.readthedocs.io/en/latest/development/ and everything seems to be fine.

Given the minimal change, it was quite surprising to me that my patch to lead to so much interaction. I hope the rebased commit works for you. Thank you for your good work! 👍